packages/components/filters/README.md

```diff
@@ -34,14 +36,193 @@ npm --save install react
 ```jsx
 import Filters from '@commercetools-uikit/filters';
 
-/**TODO: EXPAND THIS */
-const Example = () => <Filters />;
-
-export default Example;
+/**  Input for a specific filter, provided by parent application */
```

Comment:
I think that the comments are helpful for the readme, but let me know if they seem too noisy.